engine/bindings/private/audio/audio_bindings.cpp

```diff
@@ -0,0 +1,88 @@
+#include "audio_bindings.hpp"
+
+#include "audio_module.hpp"
+#include "utility/wren_entity.hpp"
+
+#include <audio_emitter_component.hpp>
+#include <audio_listener_component.hpp>
+
+#include "log.hpp"
+
+namespace bindings
+{
+void LoadBank(AudioModule& self, const std::string& path)
+{
+    BankInfo bi {};
+    bi.path = path;
+
+    self.LoadBank(bi);
+}
+
+void LoadSFX(AudioModule& self, const std::string& path, const bool is3D, const bool isLoop)
+{
+    SoundInfo si {};
+
+    si.path = path;
+    si.is3D = is3D;
+    si.isLoop = isLoop;
+
+    self.LoadSFX(si);
+}
+
+SoundInstance PlaySFX(AudioModule& self, const std::string& path, const float volume)
+{
+    if (!self.isSFXLoaded(path))
+    {
+        bblog::error("Tried to play a sound that was not loaded: {0}", path);
```

Comment:
Shouldn't you return early in this function if the sound was not loaded? Wrenbindings support the use of optionals, which means for wren you can return a SoundInstance or None

Reply:
Oh very interesting, I'll implement it. Thank you!